### PR TITLE
week5

### PR DESCRIPTION
**Pull request by AnnaViktorovna**
mongodb

### CODE REVIEW

**Comment by olearyma** [comment]
There's missing code/functionality compared to what was covered in the lesson for the Task Manager API
controller/tasks.js is missing
routes/tasks.js is missing

You should be able to use PostMan to verify the task API is working.
Create Task needs to be implemented and verify Mongo Database is created

File: 03-task-manager/starter/controllers/task.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const getAllTasks = (req, res) => {
+    res.send("get all items");
+};
+
+const createTask = (req, res) => {
```
Comment (olearyma) [inline 03-task-manager/starter/controllers/task.js:5]: Around the 1:22:51 mark, the video covers the implementation of the create task
https://www.youtube.com/watch?v=rltfdjcXjmk&t=4587s

This needs to be implemented to confirm that the MongoDB database can be created by the code.

Comment (AnnaViktorovna) [reply 03-task-manager/starter/controllers/task.js]: I think it's now work properly

**Review by olearyma — APPROVED** [review]
Nice job.  I was able to create a task and verified the collection is created in my mongodb database.

**Pull request merged**